### PR TITLE
Add Proftpd TLS configs

### PR DESCRIPTION
**Pull request by tlcd96**
Reference: #1087

### CODE REVIEW

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
Why not TLS 1.3 enabled?
http://www.proftpd.org/docs/RELEASE_NOTES-1.3.7rc1

**Comment by tlcd96** [comment]
@jaapmarcus to enable it on hestiacp ubuntu 16.04.5.
I would have to recompile it against the source for it to have the tlsv1.3 extension.
this is the supported for my vps (with the latest updates) (gives error on initialization with TLS 1.3)

Also i think that hestiacp want's the things compatible with cross servers, and if i'm not wrong, the ubuntu 16.04 it's still supported, so for now, we'll have to suck it up, until a patch comes out to enable it, or hestia will have to have a binary+extensions solely for proftpd

**Comment by tlcd96** [comment]
proof:
image
image
ps i didn't remember the version, so i googled the command for getting the version
image

File: install/deb/proftpd/tls.conf [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#
+<IfModule mod_dso.c>
+    # If mod_tls was built as a shared/DSO module, load it
+    LoadModule mod_tls.c
+</IfModule>
+<IfModule mod_tls.c>
+TLSEngine                               on
+TLSLog                                  /var/log/proftpd/tls.log
+# this is an example of protocols, proftp works witl all, but use only the most secure ones like TLSv1.1 and TLSv1.2
+TLSProtocol                             TLSv1.1 TLSv1.2
```
Comment (tlcd96) [inline install/deb/proftpd/tls.conf:16]: ubuntu 16.04.7:
if included the `TLSv1.3` proftpd fails to start with error: 
`unknown protocol: 'TLSv1.3' on line 15 of '/etc/proftpd/tls.conf'`

Comment (jaapmarcus) [reply install/deb/proftpd/tls.conf:16]: You have point 

Ubuntu 20.04 
root@test:/etc/proftpd# proftpd -v
ProFTPD Version 1.3.6c

**Comment by jaapmarcus** [comment]
Just created new install on my server

Proftd over FTP works fine
When I enable FTPTLS/SSL I got the error:
ug 19 10:05:54 test proftpd: pam_unix(proftpd:session): session opened for user admin by (uid=0)
Aug 19 10:05:54 test proftpd: pam_unix(proftpd:session): session closed for user admin

**Comment by tlcd96** [comment]
@jaapmarcus check the permissions and the log from proftpd, i'm almost certain that is a permission issue on your server (proftpd cannot access the ssl certificate due to not having permissions for the certificate)

**Comment by jaapmarcus** [comment]
Added `TLSOptions NoSessionReuseRequired` and it worked fine..

**Comment by tlcd96** [comment]
i'll, change the commit to include it

**Review by jaapmarcus — APPROVED** [review]
Still minor changes need to be made...

**Pull request merged**